pkg/network/node/networkpolicy.go

```diff
@@ -357,18 +357,23 @@ func (np *networkPolicyPlugin) generateNamespaceFlows(otx ovs.Transaction, npns
 		return
 	}
 
-	var allPodsSelected, affectsIngress, affectsEgress bool
+	var allIngressPodsSelected, allEgressPodsSelected, affectsIngress, affectsEgress bool
```

Comment:
nit: could you change the variables to `allPodsSelectedIngress` and `allPodsSelectedEgress`